reaptcha does not support react 18

Hi, when I try to install this package in a fresh project with react 18.0.0 it throws the following error:

`npm i reaptcha`

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: frontend@0.1.0
npm ERR! Found: react@18.0.0
npm ERR! node_modules/react
npm ERR!   react@"18.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.6.3 || ^17.0.0" from reaptcha@1.11.0
npm ERR! node_modules/reaptcha
npm ERR!   reaptcha@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/nvortega/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nvortega/.npm/_logs/2022-04-06T10_37_16_327Z-debug.log
```

Do you have plans to support it?

--- a/.changeset/real-sloths-grab.md
+++ b/.changeset/real-sloths-grab.md
@@ -1,0 +1,5 @@
+---
+'reaptcha': minor
+---
+
+support react@^18

--- a/.prettierrc
+++ b/.prettierrc
@@ -1,3 +1,4 @@
 {
-    "singleQuote": true
+  "singleQuote": true,
+  "trailingComma": "none"
 }

--- a/lib/index.tsx
+++ b/lib/index.tsx
@@ -51,7 +51,7 @@ export type Props = RecaptchaBaseConfig & {
   onExpire?: () => void;
   onError?: () => void;
   inject?: boolean;
-  children?: (renderProps: RenderProps) => Node;
+  children?: (renderProps: RenderProps) => ReactNode;
 };
 
 type State = {

--- a/lib/package.json
+++ b/lib/package.json
@@ -22,7 +22,7 @@
     "build": "rimraf dist && microbundle --jsx React.createElement"
   },
   "peerDependencies": {
-    "react": "^16.6.3 || ^17.0.0"
+    "react": "^16 || ^17 || ^18"
   },
   "devDependencies": {
     "microbundle": "^0.14.2",

